code/modules/shuttle/emergency.dm

```diff
@@ -129,8 +129,9 @@
 		// shuttle timers use 1/10th seconds internally
 		SSshuttle.emergency.setTimer(ENGINES_START_TIME)
 		var/system_error = emagged ? "SYSTEM ERROR:" : null
+		var/emag_boost_message = pick("STRAP IN FOR EMERGENCY BURN", "PREPARE FOR LUDICROUS SPEED", "WHEN THIS BABY HITS 88 MILLION MILES PER HOUR YOU'RE GONNA SEE SOME SERIOUS SHIT", "MAKING THE JUMP TO %@ERROR^& IN UNDER 12 PARSECS", "INERTIAL DAMPENERS OFFLINE", "WHAT IF THE ENGINES MOVE THE UNIVERSE, NOT THE SHUTTLE?")
```

Comment:
imo remove the back to the future one, and the last one

Reply:
^ Those are the only two I'd remove as well.

Reply:
those are literally the best two